interfaces/apparmor/template.go

```diff
@@ -375,6 +375,9 @@ var defaultTemplate = `
   @{PROC}/net/dev r,
   @{PROC}/@{pid}/net/dev r,
 
+  # Read-only of this snap
+  /var/lib/snapd/snaps/@{SNAP_NAME}_*.snap r,
```

Comment:
CC @bboozzoo is this ok or should we have SNAP_INSTANCE_NAME here?

Reply:
Because of how snap files are named, the rule will match both snap name and snap instance name right now. So `@{SNAP_NAME}` snap will be able to access `@{SNAP_INSTANCE_NAME}` snap files too. We do not allow `_` in snap names, so there is no chance of having a similarly named, but unrelated snap file being accessed there.

We could make the rule more explicit, and include a comment:
```
# parallel installs: this leaks access to instance snap files to @{SNAP_NAME}
 /var/lib/snapd/snaps/{@{SNAP_NAME},@{SNAP_INSTANCE_NAME}}_*.snap r,
```